tests/BuildSystem/Build/stale-file-removal.llbuild

```diff
@@ -0,0 +1,24 @@
+# Check the handling of stale file removal.
+#
+#
+# RUN: rm -rf %t.build
+# RUN: mkdir -p %t.build
+# RUN: cp %s %t.build/build.llbuild
+# RUN: %{llbuild} buildsystem build --serial --chdir %t.build > %t.initial.out
+# RUN: %{FileCheck} --input-file=%t.initial.out %s
```

Comment:
Can we extend it to cover that situation? We don't have good tools for this, but the hack I use for it right now is: https://github.com/apple/swift-llbuild/blob/c324ee3375fa9b128fe1502c74790ee99f925f9b/tests/BuildSystem/Build/signatures.llbuild#L8